daos-stack/daos
# DAOS-3629 test: Implemented list_containers.py

Test steps:
1. Create 1 container, list, and verify the listed container UUID
    matches the UUID returned when created.
2. Create 1 more container and verify list; 2 total containers
3. Create 98 more containers and verify list; 100 total containers
4. Create 2 additional pools and create 10 containers in each pool.
    Verify the container UUIDs in these 2 new pools.

Added pool_list_cont method and regex in daos_utils.py

Test-tag: pr,-hw list_containers
Signed-off-by: Makito Kano <makito.kano@intel.com>
